modules/core/src/main/scala/org/tesselation/domain/db/AddressStorage.scala

```diff
@@ -0,0 +1,11 @@
+package org.tesselation.domain.db
+
+import org.tesselation.schema.address.{Address, Balance}
+
+trait AddressStorage[F[_]] {
+  def getBalance(address: Address): F[Balance]
+
+  def updateBalance(address: Address, balance: Balance): F[schema.StoredAddress]
```

Comment:
Import schema directly

Reply:
Yet, I think it shouldn't be named "db" because it leaks the implementation detail